src/test/scala/io/iohk/ethereum/jsonrpc/EthServiceSpec.scala

```diff
@@ -205,13 +210,21 @@ class EthServiceSpec extends FlatSpec with Matchers with ScalaFutures {
     )
   }
 
+  it should "return requested work" in new TestSetup {
+    (blockGenerator.generateBlockForMining _).expects().returning(block)
+
+    val response = ethService.getWork(GetWorkRequest())
+
+    response.futureValue shouldEqual GetWorkResponse(powHash, seedHash, target)
+  }
+
```

Comment:
submit in next PR submitWork is used to include blocks into the chain, will add it in next PR